deploy-manage/security.md

```diff
@@ -75,64 +74,186 @@ $$$maintaining-audit-trail$$$
+* Manage [API keys](/deploy-manage/api-keys.md) used for programmatic access to Elastic.
+::::
 
-## Security topics
+The availability and configurability of security features vary by deployment type. On every page, you'll see deployment type indicators that show which content applies to specific deployment types. Focus on sections tagged with your deployment type and look for subsections specifically addressing your deployment model.
 
-The documentation is organized into three main areas.
+At the end of this doc, there's also a [comparison table](#comparison-table) showing feature availability and configurability by deployment type.
 
-On every page, you'll see deployment type indicators that show which content applies to specific deployment types. Focus on sections tagged with your deployment type and look for subsections specifically addressing your deployment model.
+## Managed security in Elastic Cloud
+```yaml {applies_to}
+deployment:
+  ess: all
+serverless: all
+```
 
-### 1. Secure your orchestrator
+Elastic Cloud has built-in security. For example, HTTPS communications between Elastic Cloud and the internet, as well as inter-node communications, are secured automatically, and cluster data is encrypted at rest. 
 
-The [security of your orchestrator](security/secure-hosting-environment.md) forms the foundation of your overall security posture. This section covers environment-specific security controls:
+You can augment Elastic Cloud security features in the following ways:
+* Configure [traffic filtering](./security/traffic-filtering.md) to prevent unauthorized access to your deployments. [Elastic Cloud Hosted]
+* Encrypt your deployment with a [customer-managed encryption key](./security/encrypt-deployment-with-customer-managed-encryption-key.md). [Elastic Cloud Hosted]
+* [Secure your settings](./security/secure-settings.md) using {{es}} and {{kib}} keystores. [Elastic Cloud Hosted]
+* Use the list of [Elastic Cloud static IPs](./security/elastic-cloud-static-ips.md) to allow or restrict communications in your infrastructure.
 
-- [**Elastic Cloud Hosted and Serverless**](security/secure-your-elastic-cloud-organization.md)
-- [**Elastic Cloud Enterprise**](security/secure-your-elastic-cloud-enterprise-installation.md)
-- [**Elastic Cloud on Kubernetes**](security/secure-your-eck-installation.md)
+::::{note}
+While serverless projects are fully managed and secured by Elastic, additional security settings are available for you to configure individually for your {{ech}} deployments.
+::::
 
-:::{note}
-There is no orchestration layer for self-managed deployments because you directly control the host environment. Refer to [](security/manually-configure-security-in-self-managed-cluster.md) to learn more about securing self-managed installations.
-:::
+Refer to [Elastic Cloud security](https://www.elastic.co/cloud/security) for more details about Elastic security and privacy programs.
+
+## Cluster or deployment security features
```

Comment:
I think it's a little confusing not having an orchestrator security section here / a clients + integrations section here. we can do that later.

overall this section is pretty chunky now and we can probably cut it down. again, we can ship this and clean it up later.